cmd/cloud-controller-manager/gkeservicecontroller.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	cloudprovider "k8s.io/cloud-provider"
+	"k8s.io/cloud-provider-gcp/cmd/cloud-controller-manager/options"
+	gkeservicecontroller "k8s.io/cloud-provider-gcp/pkg/controller/service"
+	"k8s.io/cloud-provider/app"
+	cloudcontrollerconfig "k8s.io/cloud-provider/app/config"
+	controllermanagerapp "k8s.io/controller-manager/app"
+	"k8s.io/controller-manager/controller"
+	"k8s.io/klog/v2"
+)
+
+type gkeServiceController struct {
+	gkeServiceControllerOptions options.GkeServiceControllerOptions
```

Comment:
why this struct has only one field, which is another struct?

Would it be okay to just use the options as the controller, or create a constructor. Does it need to be in `package main`?